js/scenes/FTU.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Image,
+  Text,
+  Button,
+  Dimensions,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import ViewPager from 'react-native-viewpager';
+import ViewPagerIndicator from '../components/ViewPagerIndicator';
+import { connect } from 'react-redux';
+import { theme } from '../../config';
+
+export class FTU extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+
+    const PAGES = [
+      this.renderPage1(),
+      this.renderPage2(),
+    ];
+    const dataSource = new ViewPager.DataSource({
+      pageHasChanged: (p1, p2) => p1 !== p2,
```

Comment:
I hate that RN doesn't have this datasource definition by default, I think we have the same filter in the home with the list view.